src/en/charms-storage.md

```diff
@@ -415,6 +430,72 @@ provider currently supports a single pool configuration attribute:
 
 ### LXD (lxd)
 
+LXD-based models have access to the 'lxd' storage provider. The LXD provider
+does not currently have any specific configuration options.
```

Comment:
The LXD provider does have configuration options: `driver` and `lxd-pool`. The former is the LXD storage driver, the latter is the name to give to the corresponding storage pool in LXD. Any other configuration you specify will be passed along to LXD when creating the LXD storage pool (e.g. zfs.pool_name in the builtin lxd-zfs pool).